typescript/src/e2e.test.ts

```diff
@@ -1542,6 +1543,1054 @@ describe("CDP Client E2E Tests", () => {
     });
   });
 
+  describe("Policy Evaluation", () => {
+    let policy: Policy;
+
+    beforeAll(async () => {
+      policy = await cdp.policies.createPolicy({
+        policy: {
+          description: Date.now().toString(),
+          scope: "account",
+          rules: [
+            {
+              action: "accept",
+              operation: "signEvmMessage",
+              criteria: [{ match: "^$", type: "evmMessage" }],
+            },
+          ],
+        },
+      });
+    });
+
+    async function cleanup() {
+      await cdp.evm.updateAccount({
+        address: testAccount.address,
+        update: {
+          accountPolicy: "",
+        },
+      });
+      await cdp.policies.deletePolicy({
+        id: policy!.id,
+      });
+    }
+
+    afterAll(cleanup);
+
+    describe("signEvmTransaction", () => {
+      describe("ethValue", () => {
+        it(">=", async () => {
+          await cdp.policies.updatePolicy({
+            id: policy.id,
+            policy: {
+              rules: [
+                {
+                  operation: "signEvmTransaction",
+                  action: "reject",
+                  criteria: [
+                    { type: "ethValue", operator: ">=", ethValue: "10000000000000000000" },
+                  ],
+                },
+              ],
+            },
+          });
+          await cdp.evm.updateAccount({
+            address: testAccount.address,
+            update: { accountPolicy: policy.id },
+          });
+          try {
+            await cdp.evm.signTransaction({
+              address: testAccount.address,
+              transaction: serializeTransaction({
+                chainId: baseSepolia.id,
+                to: "0x0000000000000000000000000000000000000000",
+                value: parseEther("11"),
+                type: "eip1559",
+                maxFeePerGas: BigInt(20000000000),
+                maxPriorityFeePerGas: BigInt(1000000000),
+                gasLimit: BigInt(21000),
+                nonce: 0,
+              }),
+            });
+            expect(true).toBe(false); // Should not reach here
```

Comment:
no big deal, but for this kind of check, i think you could use `await expect(.....).rejects.toMatchObject({
      statusCode: 403,
      errorType: "policy_violation"
    });` or `await expect(...).toThrow(...)`